crates/typst/src/geom/color.rs

```diff
@@ -27,68 +115,421 @@ use crate::syntax::Spanned;
 /// #rect(fill: aqua)
 /// #rect(fill: color.aqua)
 /// ```
-#[ty(scope)]
 #[derive(Copy, Clone, Eq, PartialEq, Hash)]
+#[ty(scope)]
 pub enum Color {
     /// An 8-bit luma color.
     Luma(LumaColor),
+
```

Comment:
Enums typically don't have blank lines in the current code style

Reply:
True, but whyyyyyyyyyy?